tests/test_Admin.py

```diff
@@ -869,10 +869,14 @@ def test_describe_user_scram_credentials_api():
     # Describe User Scram API
     a = AdminClient({"socket.timeout.ms": 10})
 
+    f = a.describe_user_scram_credentials()
```

Comment:
Add an integration test for the fix that we are deploying i.e. describe all users case.